File: Misc/CountBillion.py
import time

start_time = time.time()

# Count to 1 billion
for i in range(1000000000):
    pass

end_time = time.time()
time_taken = (end_time - start_time) * 1000  # Convert to milliseconds

print("Time taken to count to 1 billion: %.2f milliseconds" % time_taken)

File: Misc/CountBillion.java
public class CountBillion {
    public static void main(String[] args) {
        long startTime = System.currentTimeMillis();
        long k = 0;
        // Count to 1 billion
        for (double i = 0; i < 1000000000; i++) {
            // Do nothing
            k = k + 1;
        }
        System.out.println(k);
        long endTime = System.currentTimeMillis();
        long timeTaken = endTime - startTime;

        System.out.println("Time taken to count to 1 billion: " + timeTaken + " ms");
    }
}
